src/components/Accordion/Accordion.tsx

```diff
@@ -205,16 +202,20 @@ const getElementsFromData = (
  * Accordion component that shows content on toggle. Can be used to display
  * multiple accordion items together.
  */
+
+type ExtendedAccordionProps = AccordionProps &
```

Comment:
I personally prefer interfaces over types (more flexible), but don't change it. My actual comment is that this might be hard to scale if new props are needed in the future. That seems like a low risk for `Accordion`, but something to consider for other components.

Reply:
Can you also export this type and then export it in the main `index.tsx` file?

Reply:
What about this instead?

```
type AccordionButtonAttributes = Pick<
  ComponentProps<typeof AccordionButton>,
  "aria-label"
>;

export interface AccordionProps extends AccordionButtonAttributes {
...
}
```

Reply:
That looks good!